app/src/main/java/com/yagni/view/HomeGUI.java

```diff
@@ -1,136 +1,119 @@
 package com.yagni.view;
-
-import java.awt.BorderLayout;
-import java.awt.Color;
-import java.awt.Dimension;
-import java.awt.Desktop; 
-import java.awt.FlowLayout;
-import java.awt.Font;
-import java.awt.FontFormatException;
-import java.awt.GraphicsEnvironment;
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
+import java.awt.*;
+import java.awt.event.*;
+import javax.swing.*;
 import java.io.File;
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
 
-import javax.swing.ImageIcon;
-import javax.swing.JButton;
-import javax.swing.JFrame;
-import javax.swing.JLabel;
-import javax.swing.JPanel;
 
-public class HomeGUI implements ActionListener{
+public class HomeGUI implements ActionListener {
 
-    JFrame frame = new JFrame("Yagni Inc. Ordering and Inventory Management");  // Crates Frame & Title
-    JButton employeeButton = new JButton("Employee Login"); //Button to launch Employee Login
-    JButton customerButton = new JButton("Customer Ordering");     //Button to launch Customer Web Page
-    JPanel headerPanel = new JPanel();          //Creates panel for header content 
-    JPanel bodyPanel = new JPanel();            //Creates panel for body content 
-    JPanel buttonPanel = new JPanel();          //Creates inner panel for buttons in the body 
-    JPanel footerPanel = new JPanel();          //Creates panel for footer content 
-    JLabel headerLabel = new JLabel();          //text/image label for header
-    JLabel footerLabel = new JLabel();          //text label for Yagni Inc. Copyright info
-    ImageIcon logoImg = new ImageIcon("App/assets/img/YAGNI_logo.png");    //loads logo image 
+    JFrame frame = new JFrame("Yagni Inc. Ordering and Inventory Management");      // Crates Frame & Title
+    JButton employeeButton = new JButton("Employee Login");     // Button to launch Employee Login
+    JButton customerButton = new JButton("Customer Ordering");      // Button to launch Customer Web Page
+    JPanel headerPanel = new JPanel();      // Creates panel for header content
+    JPanel bodyPanel = new JPanel();        // Creates panel for body content
+    JPanel buttonPanel = new JPanel();      // Creates inner panel for buttons in the body
+    JPanel footerPanel = new JPanel();      // Creates panel for footer content
+    JLabel headerLabel = new JLabel();      // text/image label for header
+    JLabel footerLabel = new JLabel();      // text label for Yagni Inc. Copyright info
+    ImageIcon logoImg = new ImageIcon("App/assets/img/YAGNI_logo.png");     // loads logo image
 
```

Comment:
We need to update the variables to just be declared here inside the class and then initialize them inside the constructor. Additionally, all inline comments should now just be one space after the end of the statement instead of two tabs.